app/core/models/ocelot/stanzas/SequenceStanza.scala

```diff
@@ -55,6 +55,7 @@ object SequenceStanza {
 case class Sequence(text: Phrase,
```

Comment:
As we dont support a sequence with a single checkbox, think the validation should enforce "next" having minimum length 3 and "options" minimum length 2, the latter wil simplify later options handling

Reply:
Done